hera_cal/cal_formats.py

```diff
@@ -112,10 +110,31 @@ def __init__(self, meta, gains, flags=None, DELAY=False, ex_ants=[], appendhist=
         self.time_array = time
         self.integration_time = meta['inttime']
         self.gain_convention = 'divide'
+        self.set_redundant()
```

Comment:
I am fine with having this be the default value, but we should mention in the documentation that this is the case.

Reply:
ok, added it in